ons_alpha/workflows/static/workflows/js/action-readonly-review.js

```diff
@@ -0,0 +1,59 @@
+function initReviewButton() {
+    let reviewButton = document.querySelector('button[name="action-readonly-review"]');
+    if (!reviewButton) {
+        return;
+    }
+
+    const target = reviewButton.parentElement.querySelector('button[data-w-dropdown-target="toggle"]');
+    if (!target) {
+        return;
+    }
+    reviewButton.addEventListener('click', function (event) {
+        target.click();
+        event.preventDefault();
+    });
+}
+
+// Function to find the React internal instance
+function getReactInternalInstance(element) {
+    return Object.keys(element).find(key =>
+        key.startsWith('__reactInternalInstance$')
+    );
+}
+
+function preventContentChanges(editorInstance) {
+  const currentContent = editorInstance.getEditorState().getCurrentContent();
+  const originalOnChange = editorInstance.onChange;
+
+  editorInstance.onChange = (editorState) => {
+    if (editorState.getCurrentContent() !== currentContent) {
+      // Prevent content changes by reverting to the original content
+      return originalOnChange(DraftJS.EditorState.undo(editorState));
```

Comment:
this basically undoes any changes to the rich text editor since we no longer prevent pointer/selection events. Helpful for adding comments